WeatherMonitoringAndReportingService/WeatherStation/WeatherStation.cs

```diff
@@ -0,0 +1,27 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using WeatherMonitoringAndReportingService.Bots;
+using WeatherMonitoringAndReportingService.Config;
+using WeatherMonitoringAndReportingService.WeatherDetails;
+
+namespace WeatherMonitoringAndReportingService.WeatherStation
+{
+    public class WeatherStation : IWeatherStationSubject
+    {
+        private List<IBot> _bots = new();
```

Comment:
NIT: Use collection expression

```suggestion
        private List<IBot> _bots = [];
```